src/Bigtable/Connection/TableAdminConnectionInterface.php

```diff
@@ -0,0 +1,254 @@
+     *              - Tablet 3 `[customer_1, customer_2) => {"customer_1"}.`
+     *              - Tablet 4 `[customer_2, other)      => {"customer_2"}.`
+     *              - Tablet 5 `[other, )                => {"other", "zz"}.`
+     *     @type \Google\GAX\RetrySettings|array $retrySettings
+     *          Retry settings to use for this call. Can be a
+     *          {@see Google\GAX\RetrySettings} object, or an associative array
+     *          of retry settings parameters. See the documentation on
+     *          {@see Google\GAX\RetrySettings} for example usage.
+     * }
+     * @return TBD
+     *
+     */
+    public function createTableWithColumnFamily($tableId, $columnFamily, array $optionalArgs = []);
+
+    /**
+     * Permanently deletes a specified table and all of its data.
+     *
+     * @param string $tableId       The table should be deleted from the parent instance
+     *
+     * @param array  $optionalArgs {
+     *                             Optional.
+     *
+     *     @type \Google\GAX\RetrySettings|array $retrySettings
+     *          Retry settings to use for this call. Can be a
+     *          {@see Google\GAX\RetrySettings} object, or an associative array
+     *          of retry settings parameters. See the documentation on
+     *          {@see Google\GAX\RetrySettings} for example usage.
+     * }
+     *
+     *
+     * @return TBD
+     *
+     */
+    public function deleteTable($tableId, array $optionalArgs = []);
+
+    /**
+     * Lists all tables served from a specified instance.
+     *
+     * @param array  $optionalArgs {
+     *                             Optional.
+     *
+     *     @type int $view
+     *          The view to be applied to the returned tables' fields.
+     *          Defaults to `NAME_ONLY` if unspecified; no others are currently supported.
+     *          For allowed values, use constants defined on {@see \Google\Bigtable\Admin\V2\Table_View}
+     *     @type string $pageToken
+     *          A page token is used to specify a page of values to be returned.
+     *          If no page token is specified (the default), the first page
+     *          of values will be returned. Any page token used here must have
+     *          been generated by a previous call to the API.
+     *     @type \Google\GAX\RetrySettings|array $retrySettings
+     *          Retry settings to use for this call. Can be a
+     *          {@see Google\GAX\RetrySettings} object, or an associative array
+     *          of retry settings parameters. See the documentation on
+     *          {@see Google\GAX\RetrySettings} for example usage.
+     * }
+     *
+     * @return TBD
+     */
+    public function listTables(array $optionalArgs = []);
+
+    /**
+     * Gets metadata information about the specified table.
+     *
+     * @param string $tableId
+     *
+     * @return TBD
+     *
+     */
+    public function getTable($tableId);
```

Comment:
It would be ideal to define the return type for `getTable`.  The return type for `getTable` and the input type for `createTable` should be a common object, if possible.